src/main/java/com/thehrdivision/entities/Payroll.java

```diff
@@ -0,0 +1,25 @@
+package com.thehrdivision.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Entity
+@Getter
+@Setter
+public class Payroll {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    private LocalDate date; // date of what ?
+    private String report; // ?
```

Comment:
For me `LocalDate date` stands for the date of generating the payroll. 
`String report` is the name for the generated report, like: 
`Payroll for Jan Kowalski <date >, number`